src/components/gameScene/playerComp/PlayerCard.tsx

```diff
@@ -12,21 +12,23 @@ import {
   Typography,
 } from "@mui/joy";
 import { useEffect, useRef, useState } from "react";
-import defImg from '@app/assets/maxresdefault.jpg'
 import hubController from "@app/SignalR/HubController";
+import defImg from '@app/assets/maxresdefault.jpg'
```

Comment:
Please rename the file to something more readable and appropriate.

Example: DefaultUserImage.png

Reply:
Good point